docs/common/help.md

```diff
@@ -53,4 +52,3 @@ your applications and requirements.
     * <http://www.uscms.org/uscms_at_work/physics/computing/grid/index.shtml>
     * CMS Hyper News: <https://hypernews.cern.ch/HyperNews/CMS/get/osg-tier3.html>
     * CMS Twiki: <https://twiki.cern.ch/twiki/bin/viewauth/CMS/USTier3Computing>
-
```

Comment:
Hating on that trailing blank line, i see...

Reply:
Haha, that wasn't on purpose. But yeah, screw those trailing blank lines.

Reply:
Not sure if this is a bug in the preview but I see a :::console in the preview.  Is there a space or something missing which is preventing :::console from being interpreted?

Reply:
@sthapa The `:::console` (and ``````console) constructs get processed by MkDocs so it'll appear as-is in the GitHub preview